packages/osd-i18n/package.json

```diff
@@ -6,11 +6,6 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "private": true,
-  "scripts": {
-    "build": "../../scripts/use_node scripts/build",
```

Comment:
Should we be deleting the associated script that's no longer referenced here (`scripts/build`)?